sources/north-america/us/in/IndianaMap_Latest.geojson

```diff
@@ -11,7 +11,7 @@
         "icon": "https://www.arcgis.com/sharing/rest/content/items/c470d38a730b4a5d839602174ed97d5c/resources/ingov-logo-white.png",
         "url": "https://imagery.gis.in.gov/arcgis/rest/services/Imagery/BestAvailableTiles/ImageServer/exportImage?f=image&format=jpg&bbox={bbox}&imageSR={wkid}&bboxSR={wkid}&size={width},{height}&foo={proj}",
```

Comment:
[This ImageServer](https://imagery.gis.in.gov/arcgis/rest/services/Imagery/BestAvailableTiles/ImageServer) says it covers 2013 to 2017, whereas the `start_date` and `end_date` below say 2016 to 2021. [A different ImageServer](https://imagery.gis.in.gov/arcgis/rest/services/Imagery/BestAvailable/ImageServer/) says it covers 2016 to 2019.

/ref https://github.com/osmlab/editor-layer-index/issues/1660#issuecomment-1301743485

Reply:
Yeah you're right, it looks like this ImageServer doesn't include 2021:
![image](https://github.com/osmlab/editor-layer-index/assets/85302075/6a160c89-918d-42db-af2a-5f8ef7b5aaba)
[BestAvailibleTiles](https://imagery.gis.in.gov/arcgis/rest/services/Imagery/BestAvailableTiles/ImageServer)

![image](https://github.com/osmlab/editor-layer-index/assets/85302075/52bbe25f-b999-42f2-9e5b-b1c718564a0c)
[2021](https://imagery.gis.in.gov/arcgis/rest/services/Imagery/2021/ImageServer)

As for the other ImageServer mentioned, the description might be outdated. Their [AGOL open data portal](https://www.indianamap.org/datasets/INMap::best-available-imagery-of-indiana/about) suggests that this covers 2017-2021 and sure enough, it seems to cover 2021:

![image](https://github.com/osmlab/editor-layer-index/assets/85302075/3ca08a72-e943-44da-a291-b38ea59b56ec)
[BestAvailible](https://imagery.gis.in.gov/arcgis/rest/services/Imagery/BestAvailable/ImageServer)

![image](https://github.com/osmlab/editor-layer-index/assets/85302075/52bbe25f-b999-42f2-9e5b-b1c718564a0c)
[2021](https://imagery.gis.in.gov/arcgis/rest/services/Imagery/2021/ImageServer)